### PR TITLE
Cole lecture

### PR DESCRIPTION
**Pull request by CMacD12**
:new:

### What

Artificial Life Lecture - Cole MacDonald

### Why

For lecture presentation

### Where

In Student lectures directory

### CODE REVIEW

**Review by jameshughes89 — CHANGES_REQUESTED** [review]
It feels like it's missing something --- perhaps add a section at the end mentioning some other things within the area of artificial life that's been done.

**Comment by CMacD12** [comment]
> It feels like it's missing something --- perhaps add a section at the end mentioning some other things within the area of artificial life that's been done.

I added in a new section and branched out

File: .gitignore
Comment (jameshughes89) [inline .gitignore:1]: revert these changes

File: .buildinfo
Comment (jameshughes89) [inline .buildinfo:1]: remove this file.

File: site/student-lectures/Artificial-Life/artificial_life.rst [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+The main goal of artificial life is to understand the basic principles of living systems by means of a bottom-up approach, where the focus is on complex systems emerging from the interaction of simple components and rules.
+
+.. figure:: Logical_deterministic_individual-based_cellular_automata_model_of_single_species_population_growth.gif
+    :width: 500 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/Artificial_life
+
+    Logical deterministic individual-based cellular automata model of an ecosystem with one species. The model demonstrates a mechanism of S-shaped population growth.
+
+.. figure:: Logical_deterministic_individual-based_cellular_automata_model_of_interspecific_competition_for_a_single_limited_resource.gif
```
Comment (jameshughes89) [inline site/student-lectures/Artificial-Life/artificial_life.rst:26]: why a dash in with the underscores in the file names?

File: site/student-lectures/Artificial-Life/artificial_life.rst [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    :width: 500 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/Artificial_life
+
+    Logical deterministic individual-based cellular automata model of interspecific competition for a single limited resource
```
Comment (jameshughes89) [inline site/student-lectures/Artificial-Life/artificial_life.rst:31]: period at the end of the caption

File: site/student-lectures/Artificial-Life/artificial_life.rst [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    * What was expected was that the creatures would evolve to move in a way that would be similar to how animals move in the real world, utilizing legs and arms. 
+
+* Instead, as seen below, the creatures evolved slightly differently...
+
+.. figure:: youtube-video-gif-2.gif
+    :width: 300 px
+    :align: center
+    :target: https://www.youtube.com/watch?v=TaXUZfwACVE&list=PL5278ezwmoxQODgYB0hWnC0-Ob09GZGe2&index=9
+    
+    Examples of 3D creatures that evolved in Karl Sims' simulation
```
Comment (jameshughes89) [inline site/student-lectures/Artificial-Life/artificial_life.rst:103]: period.

**Review by jameshughes89 — CHANGES_REQUESTED** [review]
Make the final few changes and then it's good to go.

**Review by jameshughes89 — APPROVED** [review]

**Pull request merged**